step-templates/liquibase-run-command.json

```diff
@@ -22,7 +22,7 @@
eArguments += \"--password=`\"$liquibasePassword`\"\"\n#}\n\n# Add connection url\n$liquibaseArguments += \"--url=`\"$connectionUrl`\"\"\n\n# Set the location to where the file is\nSet-Location -Path $changeLogFile.Directory\n\n# Determine if the output variable needs to be set\nif ($liquibaseCommand.EndsWith(\"SQL\"))\n{\n\t# Add the output variable as the command name\n    $liquibaseArguments += \"--outputFile=`\"$PSScriptRoot/artifacts/$($liquibaseCommand).sql`\"\"\n    \n    # Create the folder\n    if ((Test-Path -Path \"$PSScriptRoot/artifacts\") -eq $false)\n    {\n    \tNew-Item -Path \"$PSScriptRoot/artifacts\" -ItemType \"Directory\"\n    }\n}\n\n\n# Add the additional switches\nforeach ($liquibaseSwitch in $liquibaseAdditionalSwitches)\n{\n    $liquibaseArguments += $liquibaseSwitch\n}\n\nswitch ($liquibaseCommandStyle)\n{\n\t\"legacy\"\n    {\n    \t# Add the command to execute\n\t\t$liquibaseArguments += $liquibaseCommand\n    }\n    \"modern\"\n    {\n    \t# Insert the command at the beginning\n        $liquibaseArguments = @($liquibaseCommand) + $liquibaseArguments\n    }\n}\n\n# Add command arguments\n$liquibaseArguments += $liquibaseCommandArguments\n\n# Display execution statement\n#Write-Host \"Executing the following command: $($liquibaseExecutable.FullName) $liquibaseArguments\"\n\n# Display what's going to be run\nif (![string]::IsNullOrWhitespace($liquibasePassword))\n{\n\t$liquibaseDisplayArguments = $liquibaseArguments.PSObject.Copy()\n    $arrayIndex = 0\n    for ($i = 0; $i -lt $liquibaseDisplayArguments.Count; $i++)\n    {\n    \tif ($null -ne $liquibaseDisplayArguments[$i])\n        {\n        \tif ($liquibaseDisplayArguments[$i].Contains($liquibasePassword))\n            {\n                $liquibaseDisplayArguments[$i] = $liquibaseDisplayArguments[$i].Replace($liquibasePassword, \"****\")\n        \t}\n        }\n    }\n    \n    Write-Host \"Executing the following command: $($liquibaseExecutable.FullName) $liquibaseDisplayArguments\"\n    #Write-Host \"Executing $($roundhouseExecutable.FullName) with $($roundhouseSwitches.Replace($roundhouseUserPassword, \"****\"))\"\n}\nelse\n{\n\tWrite-Host \"Executing the following command: $($liquibaseExecutable.FullName) $liquibaseArguments\"\n}\n\n\n# Redirection of stderr to stdout is done different on Windows versus Linux\nif ($IsWindows)\n{\n\t$liquibaseArguments += \"2>&1\"\n\t# Execute Liquibase\n\t& $liquibaseExecutable.FullName $liquibaseArguments\n}\n\nif ($IsLinux)\n{\n\t# Execute Liquibase\n\t& $liquibaseExecutable.FullName $liquibaseArguments 2>&1\n}\n\n# Check exit code\nif ($lastExitCode -ne 0)\n{\n\t# Fail the step\n    Write-Error \"Execution of Liquibase failed!\"\n}\n\n# Check to see if there were any files output\nif ((Test-Path -Path \"$PSScriptRoot/artifacts\") -eq $true)\n{\n\t# Loop through items\n    foreach ($item in (Get-ChildItem -Path \"$PSScriptRoot/artifacts\"))\n    {\n    \tNew-OctopusArtifact -Path $item.FullName -Name $item.Name\n    }\n}\n"
```

Comment:
Some minor feedback:

1. These commented lines can be removed:

```ps
# Add "$PSSCriptRoot/DatabaseDriver" to PATH environment variable
#                Write-Host "It is $($authDll.FullName)"

#$env:PATH += "$([IO.Path]::PathSeparator)$($authDll.FullName)"
#$driverPath = "$driverPath$([IO.Path]::PathSeparator)$($authDll.FullName)"

#Write-Host "Driver path is $($env:PATH)"
```

2. Same here with commented lines:
```ps
# Check to see if username and password were specified
#if (![string]::IsNullOrWhitespace($liquibaseUsername) -and ![string]::IsNullOrWhitespace($liquibasePassword))
#{
#	# Add Username and password
#	$liquibaseArguments += "--username=$liquibaseUsername"
#	$liquibaseArguments += "--password=`"$liquibasePassword`""
#}
```

3. More commented lines

```ps
#Write-Host "Executing the following command: $($liquibaseExecutable.FullName) $liquibaseArguments"
```

and 
```ps
#Write-Host "Executing $($roundhouseExecutable.FullName) with $($roundhouseSwitches.Replace($roundhouseUserPassword, "****"))"
```

4. Real minor, typo with `Deteremine` (should be `Determine`) :grimacing: 

```ps
# Deteremine authentication method
```